simcon_project/users/views/student_management.py

```diff
@@ -38,21 +40,20 @@ class Meta:
 @user_passes_test(is_researcher)
 def student_management(request, name="All Students"):
     # gets current researcher for use later
-    added_by = Researcher.objects.get(email=request.user)
+    researcher = Researcher.objects.filter(email=request.user.id).first()
```

Comment:
This is supposed to be user.email. I tested it and user.id does not work. We need to go through and make sure we didn't make this mistake elsewhere